SimpleWAWS/Code/CsmExtensions/CsmResourceGroupExtensions.cs

```diff
@@ -369,6 +371,13 @@ private static bool IsSimpleWaws(CsmWrapper<CsmResourceGroup> csmResourceGroup)
                 csmResourceGroup.tags != null && !csmResourceGroup.tags.ContainsKey("Bad")
                 && csmResourceGroup.tags.ContainsKey("FunctionsContainerDeployed");
         }
+        private static bool IsJenkinsResource(CsmWrapper<CsmResourceGroup> csmResourceGroup)
+        {
+            return !string.IsNullOrEmpty(csmResourceGroup.name) &&
```

Comment:
You are right. But right now its only being called from the Jenkins path, and the purpose it serves is to eliminate resourcegroups in Bad or deleting state from getting loaded. I'll work on refactoring this .